src/containers/AuthenticationPage/AuthenticationPage.js

```diff
@@ -54,14 +72,22 @@ export class AuthenticationPageComponent extends Component {
       signupError,
       submitLogin,
       submitSignup,
+      confirmError,
+      submitSinguoWithIdp,
       tab,
       sendVerificationEmailInProgress,
       sendVerificationEmailError,
       onResendVerificationEmail,
       onManageDisableScrolling,
+      showSocialLoginsForTests,
```

Comment:
The snapshot test for AuthenticationPage caused some issues because in the local environment facebook app id was in environment variables and in CI it was not. Because of this, the social login button was added to snapshot when running tests locally but in CI the page was expected to be rendered without the social login button. 

Adding extra prop feels really hacky but I'm not sure in what another way we could ensure that the page is rendered similarly locally and in CI. Also with the prop, it was possible to test both options. 

Do you have any better ideas on how to handle the snapshot test (or do we even need to keep it)?

Reply:
Yeah, I agree, this feels a bit hacky.

Jest should automatically define NODE_ENV to be 'test'.
So, try adding the Facebook app id to .env.test (maybe as empty)